src/System.Runtime/tests/System/StringsMatchingNonRandomizedHashCode.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace System.Tests
+{
+    internal static class StringsMatchingNonRandomizedHashCode
+    {
+        /// <summary>
+        /// A list of strings that all have the same hashcode (-1038491930) using the legacy non randomized string hash code algorithm
+        /// </summary>
+        internal static readonly IEnumerable<string> Data = new string[] {
```

Comment:
Is the exact same hash algorithm used on both 32-bit and 64-bit?  If yes, great.  If not, this data will likely only cause the desired collisions on one of the two.

Reply:
You're right. I had missed this point. I believe as you suggested this behavior would only cause collisions on 64bit:
https://github.com/dotnet/coreclr/blob/8ec9b34df095df032e37b4c20a7359e9df7ae0ce/src/mscorlib/src/System/String.Comparison.cs#L1015

I think I would have to now find another 100 strings to use on 32 bit machines.

Reply:
maybe it would be a good idea to add that one in a separate PR and here just only run the test for 64bit?

Reply:
your choice